lib/class-wp-theme-json-gutenberg.php

```diff
@@ -233,6 +234,7 @@ class WP_Theme_JSON_Gutenberg {
 		'padding-right'              => array( 'spacing', 'padding', 'right' ),
 		'padding-bottom'             => array( 'spacing', 'padding', 'bottom' ),
 		'padding-left'               => array( 'spacing', 'padding', 'left' ),
+		'--wp-theme-block-gap'       => array( 'spacing', 'blockGap' ),
```

Comment:
Is this CSS variable name good enough and consistent without our theme.json naming scheme? cc @nosolosw @jorgefilipecosta

Reply:
Should it be `--wp--theme--block-gap`?

Reply:
That I know of, we've used these prefixes so far:

`--wp--preset--...` => for presets
`--wp--custom--...` => for custom properties defined by the theme
`--wp--style--...` => for the link color (`--wp--style--color--link`, which is now deprecated)

If this is meant to become a style themes can target, I'd perhaps recommend `--wp--style--block-gap`. I'm not too opinionated other than not having `theme` in the variable name.

Reply:
We also need to declare this in the client: `__EXPERIMENTAL_STYLE_PROPERTY` at `packages/blocks/src/api/constants.js` so the site editor can generate it as well.